RocketControlUnitGUI/src/routes/+page.svelte

```diff
@@ -114,11 +126,13 @@
     let intervalId: any;
 
     onMount(() => {
-        intervalId = setInterval(async () => {
-            await PB.collection('Heartbeat').create({
-                message: 'heartbeat'
-            });
-        }, 5000); // 5000 milliseconds = 5 seconds
+		if (auth === true) {
+			intervalId = setInterval(async () => {
+				await PB.collection('Heartbeat').create({
+					message: 'heartbeat'
+				});
```

Comment:
Maybe show a quick skeleton alert or something (or make the state transition buttons not show up or something like that) https://www.skeleton.dev/elements/alerts so we know we're not authed in some way

Reply:
Yeah sure